.github/workflows/build.yml

```diff
@@ -157,15 +157,10 @@ jobs:
           jar,
           war,
           cssvalidator-build,
-          cssvalidator-bundle,
```

Comment:
I believe those should belong to the 1st commit, for consistency.

Reply:
> I believe those should belong to the 1st commit, for consistency.

Yeah that makes sense. Made it so